### PR TITLE
Improve documentation based on QA feedback

### PR DESCRIPTION
**Pull request by igable**
[bugzid:8472]

### CODE REVIEW

File: bin/dcm-create-server [hunk 1]
```diff
@@ -10,8 +10,8 @@ if __name__ == '__main__':
         '--productid', '-p', help="Provider's product ID such as m1.large", required=True)
     parser.add_argument('--machineimage', '-m', help='Machine image ID', required=True)
     parser.add_argument('--datacenter', '-d', help='Data Center ID', required=True)
-    parser.add_argument('--description', '-D', help='Description', required=True)
-    parser.add_argument('--name', '-n', help='Server Name', required=True)
+    parser.add_argument('--description', '-D', help='Description of the server', required=True)
+    parser.add_argument('--name', '-n', help='User friendly name of the Server', required=True)
```
Comment (newbrough) [inline bin/dcm-create-server]: is "user friendly" accurate?  this is the resource hostname, correct?  user-friendly seems more like the "description" field.

Comment (newbrough) [inline bin/dcm-create-server]: consistent capitalization

**Comment by newbrough** [comment]
looks ok

File: bin/dcm-create-server (hunk 1 above)
Comment (igable) [inline bin/dcm-create-server]: Agreed, not a great use of user friendly.

**Pull request merged**